moped-database/seeds/1602292389297_initial_seed_staging.sql

```diff
@@ -76,24 +76,24 @@ INSERT INTO public.moped_proj_funding (proj_funding_id, project_id, created_at,
 -- Data for Name: moped_proj_milestones; Type: TABLE DATA; Schema: public; Owner: moped
 --
 
```

Comment:
updated seed data for the renamed field, i thought about adding seed data for the new used ``added_by_user_id`` and ``updated_by_user_id`` fields but i wasn't sure if that was necessary